web-common/src/features/metrics-views/editor/metrics-schema.json

```diff
@@ -123,6 +123,35 @@
             "type": "string",
             "description": "Controls the formatting of this measure using a d3-format string."
           },
+          "format_d3_locale": {
+            "type": "object",
+            "description": "Defines a custom locale for d3-format.",
+            "properties": {
+              "decimal": { "type": "string", "description": "The decimal point (e.g., \".\")." },
+              "thousands": { "type": "string", "description": "The group separator (e.g., \",\")." },
+              "grouping": {
+                "type": "array",
+                "items": { "type": "integer" },
+                "description": "The array of group sizes (e.g., [3]), cycled as needed."
+              },
+              "currency": {
+                "type": "array",
+                "items": { "type": "string" },
+                "minItems": 2,
+                "maxItems": 2,
+                "description": "The currency prefix and suffix (e.g., [\"$\", \"\"])."
+              },
+              "numerals": {
+                "type": "array",
+                "items": { "type": "string" },
+                "description": "Optional array of ten strings to replace the numerals 0-9."
+              },
+              "percent": { "type": "string", "description": "Optional symbol to replace the percent suffix." },
+              "minus": { "type": "string", "description": "Optional minus sign (defaults to \"−\")." },
+              "nan": { "type": "string", "description": "Optional value for not-a-number (defaults to \"NaN\")." }
+            },
+            "required": ["decimal", "thousands", "grouping", "currency"]
```

Comment:
I don't think these should all be required. Shouldn't we be able to provide a subset?
<img width="579" alt="image" src="https://github.com/user-attachments/assets/300eb333-6936-42f9-8e69-d239a9c89ca7">

Reply:
Yeah, you are getting this mismatch between values because of the incomplete spec. Following https://d3js.org/d3-format#formatLocale `decimal`, `thousands`, and `grouping` are required fields. I can add a subset but then our definition and the d3 docs would diverge and we would have to maintain our own copy of the docs for it. I am fine either way though.

Reply:
Made the change!